autopts/bot/iut_config/zephyr.py

```diff
@@ -134,6 +134,7 @@
         },
         "test_cases": [
             'VOCS', 'VCS', 'AICS', 'IAS', 'PACS', 'ASCS', 'BAP', 'HAS', 'CSIS', 'MICP'
```

Comment:
```suggestion
            'VOCS', 'VCS', 'AICS', 'IAS', 'PACS', 'ASCS', 'BAP', 'HAS', 'CSIS', 'MICP',
```

Reply:
Missing comma prevents "overlay-le-audio.conf" to be applied

Reply:
Fixed